rest/model/patch.go

```diff
@@ -11,30 +14,45 @@ import (
 
 // APIPatch is the model to be returned by the API whenever patches are fetched.
 type APIPatch struct {
-	Id              *string       `json:"patch_id"`
-	Description     *string       `json:"description"`
-	ProjectId       *string       `json:"project_id"`
-	Branch          *string       `json:"branch"`
-	Githash         *string       `json:"git_hash"`
-	PatchNumber     int           `json:"patch_number"`
-	Author          *string       `json:"author"`
-	Version         *string       `json:"version"`
-	Status          *string       `json:"status"`
-	CreateTime      *time.Time    `json:"create_time"`
-	StartTime       *time.Time    `json:"start_time"`
-	FinishTime      *time.Time    `json:"finish_time"`
-	Variants        []*string     `json:"builds"`
-	Tasks           []*string     `json:"tasks"`
-	VariantsTasks   []VariantTask `json:"variants_tasks"`
-	Activated       bool          `json:"activated"`
-	Alias           *string       `json:"alias,omitempty"`
-	GithubPatchData githubPatch   `json:"github_patch_data,omitempty"`
+	Id                *string           `json:"patch_id"`
+	Description       *string           `json:"description"`
+	ProjectId         *string           `json:"project_id"`
+	Branch            *string           `json:"branch"`
+	Githash           *string           `json:"git_hash"`
+	PatchNumber       int               `json:"patch_number"`
+	Author            *string           `json:"author"`
+	Version           *string           `json:"version"`
+	Status            *string           `json:"status"`
+	CreateTime        *time.Time        `json:"create_time"`
+	StartTime         *time.Time        `json:"start_time"`
+	FinishTime        *time.Time        `json:"finish_time"`
+	Variants          []*string         `json:"builds"`
+	Tasks             []*string         `json:"tasks"`
+	VariantsTasks     []VariantTask     `json:"variants_tasks"`
+	Activated         bool              `json:"activated"`
+	Alias             *string           `json:"alias,omitempty"`
+	GithubPatchData   githubPatch       `json:"github_patch_data,omitempty"`
+	ModuleCodeChanges []*APIModulePatch `json:"module_code_changes"`
```

Comment:
why is `APIModulePatch` a pointer?

Reply:
APIModulePatch` shouldn't be a pointer anymore since I'm making the moduleCodeChanges gql field's contents in type Task non-nullable